samples/dependency-injection/aspnetcore/Core_7.2/SampleAutofac/Startup.cs

```diff
@@ -29,25 +29,25 @@ public void Configure(IApplicationBuilder applicationBuilder, IWebHostEnvironmen
             applicationBuilder.UseDeveloperExceptionPage();
         }
 
-        #region RequestHandling
-
-        applicationBuilder.Run(
-            handler: context =>
-            {
-                if (context.Request.Path != "/")
-                {
-                    // only handle requests at the root
-                    return Task.CompletedTask;
-                }
-                var applicationServices = applicationBuilder.ApplicationServices;
-                var endpointInstance = applicationServices.GetService<IMessageSession>();
-                var myMessage = new MyMessage();
-
-                return Task.WhenAll(
-                    endpointInstance.SendLocal(myMessage),
-                    context.Response.WriteAsync("Message sent"));
-            });
-
-        #endregion
+        // #region RequestHandling
+        //
+        // applicationBuilder.Run(
+        //     handler: context =>
+        //     {
+        //         if (context.Request.Path != "/")
+        //         {
+        //             // only handle requests at the root
+        //             return Task.CompletedTask;
+        //         }
+        //         var applicationServices = applicationBuilder.ApplicationServices;
+        //         var endpointInstance = applicationServices.GetService<IMessageSession>();
+        //         var myMessage = new MyMessage();
+        //
+        //         return Task.WhenAll(
+        //             endpointInstance.SendLocal(myMessage),
+        //             context.Response.WriteAsync("Message sent"));
+        //     });
+        //
+        // #endregion
```

Comment:
Why commented out and not removed completely?

Reply:
Totally right I'll kick it out :-)